devel/onetbb/files/patch-onetbb-older-platforms.diff

```diff
@@ -0,0 +1,487 @@
++    Copyright (c) 2005-2023 Intel Corporation
+ 
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+@@ -125,7 +125,7 @@ static const char* ittnotify_lib_name = "libittnotify.dylib";
+ 
+ 
+ #ifndef LIB_VAR_NAME
+-#if ITT_ARCH==ITT_ARCH_IA32 || ITT_ARCH==ITT_ARCH_ARM
++#if ITT_ARCH==ITT_ARCH_IA32 || ITT_ARCH==ITT_ARCH_ARM || ITT_ARCH==ITT_ARCH_PPC
+ #define LIB_VAR_NAME INTEL_LIBITTNOTIFY32
+ #else
+ #define LIB_VAR_NAME INTEL_LIBITTNOTIFY64
+diff --git a/src/tbbmalloc/def/mac32-tbbmalloc.def b/src/tbbmalloc/def/mac32-tbbmalloc.def
+new file mode 100644
+index 00000000..4dae67f9
+--- /dev/null
++++ b/src/tbbmalloc/def/mac32-tbbmalloc.def
+@@ -0,0 +1,45 @@
++# Copyright (c) 2005-2023 Intel Corporation
++#
++# Licensed under the Apache License, Version 2.0 (the "License");
++# you may not use this file except in compliance with the License.
++# You may obtain a copy of the License at
++#
++#     http://www.apache.org/licenses/LICENSE-2.0
++#
++# Unless required by applicable law or agreed to in writing, software
++# distributed under the License is distributed on an "AS IS" BASIS,
++# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
++# See the License for the specific language governing permissions and
++# limitations under the License.
++
++_scalable_calloc
++_scalable_free
++_scalable_malloc
++_scalable_realloc
++_scalable_posix_memalign
++_scalable_aligned_malloc
++_scalable_aligned_realloc
++_scalable_aligned_free
++_scalable_msize
++_scalable_allocation_mode
++_scalable_allocation_command
++___TBB_malloc_safer_aligned_msize
++___TBB_malloc_safer_aligned_realloc
++___TBB_malloc_safer_free
++___TBB_malloc_safer_msize
++___TBB_malloc_safer_realloc
++___TBB_malloc_free_definite_size
++
++/* memory pool stuff */
++__ZN3rml11pool_createElPKNS_13MemPoolPolicyE
++__ZN3rml14pool_create_v1ElPKNS_13MemPoolPolicyEPPNS_10MemoryPoolE
++__ZN3rml10pool_resetEPNS_10MemoryPoolE
++__ZN3rml12pool_destroyEPNS_10MemoryPoolE
++__ZN3rml11pool_mallocEPNS_10MemoryPoolEm
++__ZN3rml9pool_freeEPNS_10MemoryPoolEPv
++__ZN3rml12pool_reallocEPNS_10MemoryPoolEPvm
++__ZN3rml20pool_aligned_reallocEPNS_10MemoryPoolEPvmm
++__ZN3rml19pool_aligned_mallocEPNS_10MemoryPoolEmm
++__ZN3rml13pool_identifyEPv
++__ZN3rml10pool_msizeEPNS_10MemoryPoolEPv
++
+diff --git a/src/tbbmalloc/def/mac64-tbbmalloc.def b/src/tbbmalloc/def/mac64-tbbmalloc.def
+index 71253e0a..4dae67f9 100644
+--- a/src/tbbmalloc/def/mac64-tbbmalloc.def
++++ b/src/tbbmalloc/def/mac64-tbbmalloc.def
+@@ -1,4 +1,4 @@
+-# Copyright (c) 2005-2021 Intel Corporation
++# Copyright (c) 2005-2023 Intel Corporation
+ #
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+@@ -29,6 +29,7 @@ ___TBB_malloc_safer_free
+ ___TBB_malloc_safer_msize
+ ___TBB_malloc_safer_realloc
+ ___TBB_malloc_free_definite_size
++
```

Comment:
No changes in this file other than this. Should probably be removed.

Reply:
Indeed, it is inconsequential and unneeded in our patch.